src/VirtoCommerce.NotificationsModule.Core/Model/SmsNotification.cs

```diff
@@ -33,7 +33,7 @@ public override async Task ToMessageAsync(NotificationMessage message, INotifica
             await base.ToMessageAsync(message, render);
 
             var smsNotificationMessage = (SmsNotificationMessage)message;
-            var template = (SmsNotificationTemplate)Templates.FindWithLanguage(message.LanguageCode);
+            var template = (SmsNotificationTemplate)Templates.FindTemplateForTheLanguage(message.LanguageCode);
```

Comment:
`The` article is not used in the names of the methods

Reply:
Fixed